agent-samples/azure/AzureOpenAIAssistants.yaml

```
@@ -1,9 +1,9 @@
 kind: Prompt
 name: Assistant
 description: Helpful assistant
-instructions: You are a helpful assistant. You answer questions is the language specified by the user. You return your answers in a JSON format. You must include Assistants as the type in your response.
+instructions: You are a helpful assistant. You answer questions in the language specified by the user. You return your answers in a JSON format. You must include Assistants as the type in your response.
 model:
-    id: =Env.AZURE_OPENAI_DEPLOYMENT_NAME
+    id: gpt-4o-mini
     provider: AzureOpenAI
     apiType: Assistants
```

I see that type is `string` and not `String`.

Are those `apiTypes` part of a enumeration or index of options, I would rather have all those indexable things as lowercase.

```suggestion
    apiType: assistants
```